xarray/tests/test_dask.py

```diff
@@ -1180,6 +1180,36 @@ def test_map_blocks_ds_transformations(func, map_ds):
     assert_identical(actual, func(map_ds))
 
 
+@pytest.mark.parametrize("obj", [make_da(), make_ds()])
```

Comment:
How about adding a cross test, where a Dataset is converted to a DataArray or the other way around?